ttnn/cpp/ttnn/operations/transformer/sdpa/device/kernels/dataflow/reader_interleaved.cpp

```diff
@@ -76,13 +99,27 @@ void kernel_main() {
     const InterleavedAddrGenFast<is_dram> mask_reader = {
         .bank_base_address = mask_addr, .page_size = mask_tile_bytes, .data_format = mask_data_format};
 
+    volatile tt_l1_ptr uint32_t* page_table_ptr;
+
     uint32_t q_tile_id = 0;
     uint32_t k_tile_id = 0;
     uint32_t v_tile_id = 0;
     uint32_t mask_tile_id = 0;
     uint32_t barrier_count = 0;
 
     for (uint32_t nb = local_batch_start; nb < local_batch_end; ++nb) {
+        if constexpr (is_chunked) {
```

Comment:
Note that the page table is read inside the loop over batch since page table varies by user